includes/class-mailer.php

```diff
@@ -87,7 +87,11 @@ public static function comment_notification_text( $message, $comment_id ) {
 		}
 
 		$post                  = \get_post( $comment->comment_post_ID );
-		$comment_author_domain = \gethostbyaddr( $comment->comment_author_IP );
+		$comment_author_domain = '';
+		// Only attempt to resolve hostname if we have a valid IP address.
+		if ( ! empty( $comment->comment_author_IP ) && \filter_var( $comment->comment_author_IP, FILTER_VALIDATE_IP ) ) {
```

Comment:
```suggestion
		if ( filter_var( $comment->comment_author_IP, FILTER_VALIDATE_IP ) ) {
```
Presumably the `filter_var()` check on its own is enough? Not sure if there's a speed concern where we'd want to avoid calling it when there's an empty value.